weave-js/src/components/Panel2/KeyValTable.tsx

```diff
@@ -1,24 +1,45 @@
 import * as globals from '@wandb/weave/common/css/globals.styles';
 import styled from 'styled-components';
 
-export const Table = styled.table`
+export const Table = styled.div`
   font-size: 13px;
+  display: table;
+  padding: 2px;
 `;
-export const Row = styled.tr``;
-export const Key = styled.td`
+Table.displayName = 'S.Table';
+
+export const Rows = styled.div`
+  margin-top: 2px;
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+`;
+Rows.displayName = 'S.Rows';
+
+export const Row = styled.div`
+  display: flex;
+  gap: 2px;
+`;
+Row.displayName = 'S.Row';
+
+export const Key = styled.div`
   white-space: nowrap;
   text-overflow: ellipsis;
   vertical-align: top;
   padding: 0 !important;
   color: ${globals.gray500};
   width: 100px;
 `;
-export const Val = styled.td`
+Key.displayName = 'S.Key';
+
+export const Val = styled.div`
   padding: 0 !important;
 `;
+Val.displayName = 'S.Val';
 
-export const InputUpdateLink = styled.div`
+export const InputUpdateLink = styled.span`
```

Comment:
This is a small functional change - it makes it so you only get the cursor and click behavior if you are over the actual text, rather than the entire "table cell", which I think is less confusing given that there is no visual indication such as a border or background for the cell.